src/tangojs-connector-mtango/MTangoConnector.js

```diff
@@ -274,11 +285,13 @@ export class MTangoConnector extends tangojs.Connector {
    */
   device_command_inout (devname, cmdname, argin) {
 
-    // PUT http://localhost:8080/mtango/rest/rc2/devices/sys/tg_test/1/commands/State?input
+    // PUT http://localhost:8080/mtango/rest/rc3/hosts/localhost/10000/devices/sys/tg_test/1/commands/State?input
     // FIXME handle result
 
     const input = argin ? `?input=${argin.value}` : ''
-    return this._fetch('put', `devices/${devname}/commands/${cmdname}${input}`)
```

Comment:
Rebase this PR on latest master and pass body (`input`?) as third argument to `_fetch`.
Note that response should be parsed and converted to `DeviceData` object.